src/data/markdown/docs/02 javascript api/09 k6-net-grpc/20 Client/30-Client-invokerpc-url-request-params.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Client.invoke(url, request [,params])"
+---
+
+Invokes an unary RPC request to the given method.
+
+The given method to invoke must have its RPC schema previously loaded via the [Client.load()](/javascript-api/k6-net-grpc/client/client-load-importpaths----protofiles) function, otherwise an
+error will be thrown.
+
+[Client.connect()](/javascript-api/k6-net-grpc/client/client-connect-address-params) must be called first before invoking a request, otherwise an error will be thrown.
```

Comment:
In both cases we should specify the actual error that is thrown instead of "an error".

Reply:
hmm we really don't have a set of predefined JS exceptions people can catch :disappointed: so, I am not sure if we _can_ specify the error at this point, without making code changes in k6... given the beta nature of the gRPC API, this is probably good enough for now